packages/core/core/src/loaders/admin.ts

```diff
@@ -1,15 +1,29 @@
 import _ from 'lodash';
-import type { Strapi } from '@strapi/types';
+import type { Strapi, Schema } from '@strapi/types';
+import { getGlobalId } from '../domain/content-type';
 
 export default async function loadAdmin(strapi: Strapi) {
   strapi.admin = require('@strapi/admin/strapi-server');
 
   strapi.get('services').add(`admin::`, strapi.admin?.services);
   strapi.get('controllers').add(`admin::`, strapi.admin?.controllers);
-  strapi.get('content-types').add(`admin::`, strapi.admin?.contentTypes);
+  strapi.get('content-types').add(`admin::`, formatContentTypes(strapi.admin?.contentTypes ?? {}));
   strapi.get('policies').add(`admin::`, strapi.admin?.policies);
   strapi.get('middlewares').add(`admin::`, strapi.admin?.middlewares);
 
   const userAdminConfig = strapi.config.get('admin');
   strapi.get('config').set('admin', _.merge(strapi.admin?.config, userAdminConfig));
 }
+
+const formatContentTypes = (contentTypes: Record<string, { schema: Schema.ContentType }>) => {
+  Object.values(contentTypes).forEach((definition) => {
+    const { schema } = definition;
+
+    Object.assign(schema, {
+      plugin: 'admin',
```

Comment:
Was this already the case before? I'm discovering this

Reply:
ha yes 100% just moving the code but not changing anything :)